test/test.cc

```diff
@@ -2288,6 +2288,8 @@ class ServerTest : public ::testing::Test {
   {
 #ifdef CPPHTTPLIB_OPENSSL_SUPPORT
     cli_.enable_server_certificate_verification(false);
+#else
+#error no ssl
```

Comment:
We shouldn't do this, since there is need to test on environment which don't support OpenSSL.